web/src/overlays.ts

```diff
@@ -239,9 +258,9 @@ export function close_overlay(name) {
 
     blueslip.debug("close overlay: " + name);
 
-    $active_overlay.removeClass("show");
+    $active_overlay?.removeClass("show");
 
-    $active_overlay.attr("aria-hidden", "true");
+    $active_overlay?.attr("aria-hidden", "true");
```

Comment:
The checks for undefined in this function can probably also all be removed if we add the type as suggested in the previous commit.